installer/bundle/bundle_skel.sh

```diff
@@ -784,15 +767,19 @@ case "$installMode" in
         if [ $scx_installed -ne 0 -a $omi_installed -ne 0 ]; then
             echo "Installing OMS agent ..."
 
-            pkg_add_list $OMI_PKG omi
-            pkg_add_list $SCX_PKG scx
             pkg_add_list $OMS_PKG omsagent
 
             python_ctypes_installed
             if [ $? -eq 0 ]; then
                 pkg_add_list $DSC_PKG omsconfig
             fi
 
+            # Install SCX (and OMI)
+            [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
+            [ -n "${debugMode}" ] && DEBUG="--debug" || DEBUG=""
+            ./bundles/${SCX_INSTALLER} --install $FORCE $DEBUG
+            SCX_STATUS=$?
+
             # Now actually install of the "queued" packages
             if [ -n "${add_list}" ]; then
                 if [ "$INSTALLER" = "DPKG" ]; then
```

Comment:
Adding Jeff's comment from the email thread:

This is a good point, and I didn’t want this to be lost. I’ve seen this too in the past, typically on Ubuntu, but I haven’t tried to replicate lately.

I think, perhaps due to bundling of installations, perhaps for some other reasons, Ubuntu can try and start services before the packages are fully configured, causing errors. This, if memory serves, appeared to be Ubuntu-specific (not on CentOS, for example).

We should definitely test these cases and make sure that sort of thing doesn’t happen in installation/upgrade scenarios.